src/forge/actors/generator.py

```diff
@@ -508,16 +508,16 @@ async def shutdown(  # pyright: ignore[reportIncompatibleMethodOverride]
         await stop_proc_mesh(actor._generator_proc)
 
     @endpoint
-    async def _test_save_model_params(self):
-        """Save model parameters before weight update, used for tesing purposes only."""
-        logger.info("[Generator] save model parameters for testing.")
-        await self.generator_worker._test_save_model_params.call()
+    async def save_model_params(self):
+        """Save model parameters before weight update, used for debugging purposes."""
```

Comment:
Nit: Can we bump the for debugging purpose to the front? It's easy to miss at the end of a line

```suggestion
        """(For Debugging/Testing): Save model parameters before weight update."""
```